templates/answer.html

```diff
@@ -0,0 +1,11 @@
+{% extends "layouts/page.html" %}
+
+{% block title %}{{ post_data['title'] }}{% endblock %}
+{% block context %}
+<p>ID: {{ post_data['id'] }}</p>
+    <p><a href="/post/{{ post_data['post_id'] }}">Post</a></p>
+<p>User ID: {{ post_data['user_id'] }}</p>
+<p>Code: {{ post_data['code'] }}</p>
```

Comment:
This should be escaped with `|e` to avoid XSS, no?

Reply:
Could you do that? IDK templating.

Reply:
just like change `{{ post_data['code'] }}` to `{{ post_data['code']|e }}`

Reply:
Alright

Reply:
Alright